object_detection/yolov3_tensorflow/main.py

```diff
@@ -0,0 +1,94 @@
+import  cv2
+import os
+from matplotlib import pyplot as plt
+from model import *
+from utils import *
+import os
+import time
+import logging
+import argparse
+import numpy as np
+import random
+from numpy import expand_dims
+from keras.preprocessing.image import load_img, img_to_array
+import tensorflow as tf
+
+
+def main():
+
+    model = make_yolov3_model()
+    # load the model weights
+    weight_reader = WeightReader('yolov3.weights')
+
+    # set the model weights into the model
+    weight_reader.load_weights(model)
+
+    # save the model to file
+    model.save('model.h5')
+
+    from keras.models import load_model
+    model = load_model('model.h5')
+
+    model.summary()
+
+    anchors = [[116,90, 156,198, 373,326], [30,61, 62,45, 59,119], [10,13, 16,30, 33,23]]
+
+    # define the expected input shape for the model
+    WIDTH, HEIGHT = 416, 416
+
+    # define the probability threshold for detected objects
+    class_threshold = 0.5
+
+    
+    images=os.listdir('images')
+
+    for file in images:
+    
+        if file == '.DS_Store':
+            continue
```

Comment:
it would be better to change condition such that, if file is a valid image file then perform the instructions. You are only filtering .DS_Store file with this